src/mmrelay/db_utils.py

```
@@ -229,70 +398,105 @@ def delete_plugin_data(plugin_name, meshtastic_id):
-                ),
-            )
-            result = cursor.fetchone()
-        try:
-            return json.loads(result[0] if result else "[]")
-        except (json.JSONDecodeError, TypeError) as e:
-            logger.error(
-                f"Failed to decode JSON data for plugin {plugin_name}, node {meshtastic_id}: {e}"
-            )
-            return []
+        result = manager.run_sync(_fetch)
     except (MemoryError, sqlite3.Error) as e:
```

Ruff (0.14.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)